airflow/www/views.py

```diff
@@ -347,47 +356,48 @@ def task_stats(self, session=None):
             )
             .join(Dag, Dag.dag_id == DagRun.dag_id)
             .filter(DagRun.state != State.RUNNING, Dag.is_active)
+            .filter(DagRun.dag_id.in_(filter_dag_ids) if selected_dag_ids else True)
```

Comment:
Instead of `else True`

we should do something like:

```python
if selected_dag_ids:
    LastDagRun = LastDagRun.filter(DagRun.dag_id.in_(filter_dag_ids))`
```

similar for RunningDagRun.

Reply:
Cheers fixed